src/main/cluster/manager.ts

```diff
@@ -239,7 +241,16 @@ export class ClusterManager {
       if (!cluster.disconnected) {
         cluster.online = false;
         cluster.accessible = false;
-        cluster.refreshConnectionStatus().catch((e) => e);
+
+        void (async () => {
+          try {
+            await this.dependencies
+              .getClusterConnection(cluster)
+              .refreshConnectionStatus();
+          } catch {
+            // ignore
+          }
+        })();
```

Comment:
Weird construction. Could we make `async onNetworkOffline` and `await` at first level function's body?
And actually why `await` needed here at all when in `catch()` nothing anyway?